docs/processors/standalone/wasm.mdx

```diff
@@ -0,0 +1,259 @@
+---
+sidebar_position: 2
+---
+
+# How it works
+
+Here we will describe how Conduit runs standalone processors under the hood.
+This is useful to understand when writing a processor SDK.
+
+## Why WebAssembly?
```

Comment:
Making this suggestion to be consistent with we express these in our docs (e.g.: "How it works" vs "How does it work?")

```suggestion
## Why WebAssembly
```

Reply:
I simplified this header and renamed it to `WebAssembly` 😉